packages/binding-firestore/src/firestore-handler.ts

```diff
@@ -74,7 +74,7 @@ export const writeDataToFirestore = async (
     content: Content,
     reqId: string = null
 ): Promise<void> => {
-    console.debug("[debug] writeDataToFirestore topic:", topic, " value:", content, reqId);
+    console.debug("[debug] writeDataToFirestore topic:", topic, reqId);
```

Comment:
We seemed to have missed it before but we usually add the "protocol" so that we know where the log comes from and we can filter it..

e.g., `console.debug("[binding-http]", `HttpServer ...`);`

Do you think we can do this for firestore too?

Reply:
@danielpeintner 

Thank you for giving comment !

> Looks good but I added a logging comment and wanted to ask you about resolving warnings like
"Unexpected any. Specify a different type".

How can I check the above message?

> Do you think we can do this for firestore too?

OK. I will modify the log to output `[binding-firestore]`.

Reply:
> > Looks good but I added a logging comment and wanted to ask you about resolving warnings like
> > "Unexpected any. Specify a different type".
> 
> How can I check the above message?

Visual Studio code should show you these warnings.
Besides that, running `npm run lint` in the root folder or for example in `packages/binding-firestore` will show you these warnings as well.
Moreover, the "Files changed" tab of Github highlights the same issues

> > Do you think we can do this for firestore too?
> 
> OK. I will modify the log to output `[binding-firestore]`.

👍

Reply:
W.r.t. debugging.

We are about to change debug module for logging.
see https://github.com/eclipse/thingweb.node-wot/pull/783/